features/external_authorization.md

```diff
@@ -154,48 +154,48 @@ N/A. Design doc presented instead.
 
 **Design**
 
-- [ ] Design doc describing the intention of the feature, how it will be
+- [x] Design doc describing the intention of the feature, how it will be
 	implemented, and any thoughts on how to test the feature has been approved by
 	relevant work group leads
-- [ ] Feature coverage and test plans written and approved.
+- [x] Feature coverage and test plans written and approved.
 
 **Docs** 
 
-- [ ] Documentation on istio.io includes performance expectations; may have caveats. 
-- [ ] Documentation on istio.io includes samples/tutorials. 
-- [ ] Documentation on istio.io includes appropriate glossary entries. 
-- [ ] All new documentation containing user actions includes istio.io tests.
+- [x] Documentation on istio.io includes performance expectations; may have caveats. 
+- [x] Documentation on istio.io includes samples/tutorials. 
```

Comment:
Correct links?
```suggestion
- [x] Documentation on istio.io includes samples/tutorials. [Here](https://istio.io/latest/docs/tasks/security/authorization/authz-custom/)
and [Here](https://istio.io/latest/blog/2021/better-external-authz/).
```

Reply:
While updating the header, I would maybe tweak/sync the names in the header and features.yaml and add a short description.

Reply:
Added link to the task.